sdk/core/Microsoft.Azure.Core.Spatial/src/Microsoft.Azure.Core.Spatial.props

```diff
@@ -0,0 +1,6 @@
+<Project>
+  <ItemGroup>
+    <!-- TODO: Convert this to a PackageReference when Microsoft.Azure.Core.Spatial is published -->
```

Comment:
Probably worth filing an issue for this follow-up if you haven't already.

Reply:
#15542